arch/arm64/boot/dts/rockchip/rk3588-firefly-itx-3588j.dts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2021 Rockchip Electronics Co., Ltd.
+ *
+ */
+
+/dts-v1/;
+
+#include "rk3588-firefly-itx-3588j.dtsi"
+//#include "rk3588-firefly-demo.dtsi"
+#include "rk3588-firefly-itx-cam-8ms1m.dtsi"
+#include "rk3588-firefly-itx-cam-2ms2m.dtsi"
+
+/ {
+	model = "Firefly ITX-3588J HDMI(Linux)";
+	compatible = "rockchip,rk3588-firefly-itx-3588j", "rockchip,rk3588";
+};
+
+/ {
+	chosen {
+	status = "disabled";
+	};
```

Comment:
We usually delete chosen like this: https://github.com/armbian/linux-rockchip/blob/rk-6.1-rkr1/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts#L27
Did you test it?

Reply:
I'm currently building, apologize

Reply:
I tested, it works fine, fan are fine too